config/yurtadm/cloudimage/openyurt-latest/Kubefile

```diff
@@ -0,0 +1,16 @@
+FROM kubernetes:v1.19.8-alpine
+
+# flannel: https://github.com/sealerio/applications/tree/main/flannel
+COPY flannel/cni .
+COPY flannel/init-kube.sh /scripts/
+COPY flannel/kube-flannel.yml manifests/
+
+COPY shell-plugin.yaml plugins
```

Comment:
when do we apply this yaml file(shell-plugin.yaml)?

Reply:
It is sealer's plugin: http://sealer.cool/docs/getting-started/plugin.html#plugin-type-list